packages/usage-examples/src/pay-eth-request.ts

```diff
@@ -0,0 +1,22 @@
+import { hasSufficientFunds, payRequest } from '@requestnetwork/payment-processor';
+import { RequestNetwork } from '@requestnetwork/request-client.js';
+import { Wallet } from 'ethers';
+
+/* tslint:disable:no-floating-promises */
+
+// for demo purpose
+const wallet = Wallet.createRandom();
+const requestNetwork = new RequestNetwork();
+
+// tslint:disable-next-line: typedef
+(async () => {
+  const account = wallet.address;
+
+  const request = await requestNetwork.fromRequestId('REQUEST_ID');
+  const requestData = request.getData();
+  if (!(await hasSufficientFunds(requestData, account))) {
+    throw new Error('You do not have enough funds to pay this request');
+  }
+  const tx = await payRequest(requestData, wallet);
```

Comment:
`const tx = await payRequest(requestData, wallet);`
Looks very nice ! :+1: 

:warning: I think it is mandatory to let the user choosing the amount he/she wants to pay!

Reply:
Why not integrating this in the request-client.js : 
`request.pay(amount, wallet)`

(for later I would say)
(taking in account that more currency = more code)